lib/Mojo/Message.pm

```diff
@@ -553,7 +553,7 @@ Check if message parser/generator is finished.
   my $bool = $msg->is_limit_exceeded;
 
 Check if message has exceeded L</"max_line_size">, L</"max_message_size">, L<Mojo::Content/"max_buffer_size"> or
-L<Mojo::Headers/"max_line_size">.
+L<Mojo::Headers/"max_line_size">. L</"max_message_size"> may be set by L<Mojolicious/"max_request_size">.
```

Comment:
That's a very random reference that doesn't really add value.

Reply:
My intent was to tie max_request_size and is_limit_exceeded together, such that someone googling "mojolicious max_request_size" has a good chance of figuring out that they have to manually check (rather than assuming that setting max_request_size is fire and forget), but it could probably work just as well in reverse by explaining at max_request_size that it sets max_message_size.